DevExtreme CLI: add React Next.js info (#7295)
DevExpress/devextreme-documentation: concepts/50 React Components/50 Application Template/10 Layouts.md

> The application template includes two layouts. The only difference between them is the toolbar type: outer (default) or inner.

Reviewer: ```suggestion
The application template has two layouts that differ only in toolbar type: outer (default) or inner.
```

> The application template includes two layouts. The only difference between them is what type of toolbar is implemented: outer (default) or inner.

Reviewer 2: It seems that a lot of content uses terms "outer" and "inner". My first idea was to change it to "external" and "internal". You could even say "difference between them is the toolbar location: outside or inside". But I'm not going to make those changes, since I see that some API uses those terms too. 

```suggestion
The application template includes two layouts. The only difference between them is the toolbar type: outer (default) or inner.
```

Author: Yeah the layouts are named "Inner" and "Outer" in the template, so I kept those names as is.
